docs/resources/Channel.md

```diff
@@ -22,14 +22,22 @@ Guild channels represent an isolated set of users and messages within a Guild.
 | bitrate | integer | the bitrate (in bits) of the voice channel | Voice only |
 | user\_limit | integer | the user limit of the voice channel | Voice only |
```

Comment:
imho ditch the guild vs dm channel distinction and call it `Channel Object`.. dont forget to change `recipient` to `recipients`